packages/core/lib/std.js

```diff
@@ -305,6 +418,7 @@ module.exports = ({
         return evalRuleSet(ruleset);
       }
     },
+    //* Interpret ruleset as array
```

Comment:
so we still need to write these correct?

Reply:
Yea, this is testing a feature where, for shorthand, you can just put text and it will be interpreted as a 'default' tag (@desc in this case). This allows for simple one line comments in some places instead of a block comment with a tag